helm-bootable.yaml

```diff
@@ -0,0 +1,9 @@
+image:
```

Comment:
I don't think we need that file. 

There is nothing specific to Bootable Jar in this configuration file except you named the application image differently.

I think your example has more strength if we show that the "packaging" of the application (with wildfly-maven-plugin or wildfly-jar-maven-plugin)  does not impact the application image and the way WildFly application are deployed on K8S.

Reply:
Ok, using same image name.